conda.recipe/meta.yaml

```diff
@@ -24,8 +24,12 @@ requirements:
     - python >=3.6
     - pip
   run:
+    - bokeh
```

Comment:
@nrbgt Lukas and I were discussing this. What dependencies are required in the conda build? All of them?

Reply:
How do you know when a dependency in the AP file should go in here?